api/pkg/imagebuilder/config.go

```diff
@@ -38,7 +38,7 @@ type Config struct {
 	// Kaniko kubernetes service account
 	KanikoServiceAccount string
 	// Kubernetes resource request and limits for kaniko
-	Resources cfg.ResourceRequestsLimits
+	DefaultResource cfg.ResourceRequestsLimits
```

Comment:
@ariefrahmansyah  just to confirm, this can be freely changed (as seen in the e2e) because the helm will simply use the whole yaml block as payload -> https://github.com/caraml-dev/helm-charts/blob/main/charts/merlin/templates/_helpers.tpl#L274

Reply:
Yes, correct.